tests/jobs/overburden_timeshift/test_ots.py

```diff
@@ -125,32 +111,42 @@ def test_create(setUp):
             parms.above,
             parms.velocity_model,
         )
+    assert str(err.value) == "[Errno 2] No such file or directory"
```

Comment:
Isn't there a better way to compare it?

Reply:
Can do like this instead: 
```
    with pytest.raises(IOError, match="No such file or directory"):
        OverburdenTimeshift(
            parms.eclbase,
            parms.mapaxes,
            parms.seabed,
            parms.youngs,
            parms.poisson,
            parms.rfactor,
            parms.convention,
            parms.above,
            parms.velocity_model,
        )
```
Or were you thinking of something else?

Reply:
I haven't thought about that but it looks even better!